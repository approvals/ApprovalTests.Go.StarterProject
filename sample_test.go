package starterpackage

import (
	approvals "github.com/approvals/go-approval-tests"
	"testing"
)

func TestNormalTest(t *testing.T) {
	if 5 != 5 {
		t.Fatal("5 != 5")
	}
}

func TestBasicApproval(t *testing.T) {
	approvals.VerifyString(t, "Hello Approvals")
}

func TestJsonApproval(t *testing.T) {
	person := Person{
		"John Galt", 100,
	}
	approvals.VerifyJSONStruct(t, person)
}
