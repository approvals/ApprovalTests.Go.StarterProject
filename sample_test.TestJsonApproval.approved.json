{
  "Name": "John Galt",
  "Age": 100
}